dts/bindings/led/nordic,npm1300-led.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Nordic nPM1300 LED controller
+
+  The nPM1300 has three LED outputs.
+  Each LED can automatically display error or charging status,
+  or be controlled by software.
+
+compatible: "nordic,npm1300-led"
+
+properties:
+  nordic,led0-mode:
+    type: int
+    description: LED 0 mode
```

Comment:
suggestion here: maybe use a string enum, and use the ENUM_IDX() macros?

e.g. `nordic,led0-mode = "charging";`

Reply:
Nice.